ex2.yml

```diff
@@ -4,8 +4,7 @@ alpha: 0.125
 global_size: { height: 60, width: 12 }
 # degree of parallelism (number of blocks in each dimension)
 parallelism: { height: 1, width: 1 }
-# PDI configuration
+
 pdi:
-  plugins:
-    trace:
-      logging: { pattern: '[PDI][%n-plugin] *** %l: %v' }
+#*** PDI configuration: use the trace plugin to print all the PDI activities
```

Comment:
Missing mention of logging in trace wiki page.

Reply:
This issuer will solved this problem https://github.com/pdidev/pdi/issues/499